crates/xline-client/examples/lock.rs

```diff
@@ -13,15 +13,17 @@ async fn main() -> Result<()> {
         .await?
         .lock_client();
 
-    // acquire a lock
-    let resp = client.lock(LockRequest::new("lock-test")).await?;
+    // create a session
+    let session = Session::new(client).build().await?;
```

Comment:
The usage it too verbose. Can you provide a function to get a mutex from client directly?

Reply:
Done, I move the session creation into the `Xutex::new`.